statvar_imports/usda/agriculture/usda_psd_stat_vars.mcf

```diff
@@ -0,0 +1,54 @@
+# Auto generated using command: "stat_var_processor.py --input_data=/usr/local/google/home/bipina/USDA/data/statvar_imports/usda/agriculture/input_data.csv --config=/usr/local/google/home/bipina/USDA/config.py --pv_map=https://docs.google.com/spreadsheets/d/1FDQz1XM8B8PdsG-4wDksS1LI8K6W66ikAkEZgy4bR-o/edit?resourcekey=0-B4D-6Olxl9s3V3BzJpym1w&gid=0#gid=0 --output_path=/usr/local/google/home/bipina/USDA/data/statvar_imports/usda/agriculture/usda_psd" on 2024-07-09 19:16:41.915847
```

Comment:
Could you please make a proper folder structure like other PR's. Please follow the steps to PR raising https://docs.google.com/spreadsheets/d/1rF6yxSjQLgkYpRiVP642B0EQbmPNqN1cwZU4_7xZZ_U/edit?resourcekey=0-Ft6LSNDQLXQ_9W-stGoz9A&gid=1086229089#gid=1086229089